spaceship/lib/spaceship/tunes/language_converter.rb

```diff
@@ -8,6 +8,14 @@ def from_itc_to_standard(from)
           (result || {}).fetch('locale', nil)
         end
 
+        # Converts the Language "UK English" to itc locale en-GB
+        def from_standard_to_itc_locale(from)
+          result = mapping.find { |a| a['name'] == from }
+          itc_locale = (result || {}).fetch('itc_locale', nil)
+          return itc_locale if itc_locale
+          (result || {}).fetch('locale', nil)
```

Comment:
I'd like to move away from using `fetch` to look up items from hashes. I find its behavior confusing relative to people's expectations about how it works.

Does the following code do the job?

```ruby
result = mapping.find { |a| a['name'] == from } || {}
return result['itc_locale'] || result['locale']
```